backend/tracim_backend/tests/functional/test_frontend.py

```diff
@@ -51,7 +51,7 @@ def test_api__csp_header__ok_200__additional_directives(self, web_testapp):
     def test_api__csp_header__ok_200__with_collabora(self, web_testapp):
         res = web_testapp.get("/", status=200)
         csp = res.headers.get("Content-Security-Policy")
-        assert "frame-src http://localhost:9980" in csp
+        assert "frame-src *" in csp
```

Comment:
The result should be `* http://localhost:9980`. Not that it changes anything here, but just to ensure that the mechanism for backend applications to tune the CSP is effective.

Reply:
I hesitated, but let's be explicit indeed

```suggestion
        assert "frame-src * http://localhost:9980" in csp
```